en/docs/guides/user-mgt/managing-user-attributes.md

```diff
@@ -0,0 +1,204 @@
+# Managing User Attributes
+
+When using the WSO2 Identity Server (WSO2 IS) for user and role management, it is
+important to understand how to manage the attributes of users within it.
+In WSO2 IS, each user store attribute can be mapped as a
+claim. Therefore, you need to use the claim management functionality
+available in WSO2 IS to properly map your LDAP/AD/JDBC user
+store attributes with the claim URIs defined by WSO2 IS. You
+can also add different claim URIs and manage them using claim
+management.
+
+The following topics provide instructions on how to manage user
+attributes in WSO2 IS.
+
+## Managing the attributes of a user
+
+The following are the three main ways to view, add, edit, and delete
+attributes of a user.
+
+1.  By accessing the profile of the user and changing the attributes
+    using the [admin Portal](link-admin-portal)
+2.  You can use the REST Web service according to the SCIM provisioning specification. For more information on this, see [Using the SCIM 2.0 REST APIs](link-using-the-scim-2.0-rest-apis).
+3.  You can use the `           RemoteUserStoreManagerService          `
+    API. This is a SOAP-based API and is very easy to use. For more
+    information on using this, see [Managing Users and Roles with
+    APIs](insert-link). Supposing you want to
+    set a user attribute, you can call the following method.
```

Comment:
Supposing you want to set a user attribute, you can call the following method --> Call the following method to set a user attribute